reference/paths/transactions__post-approve-transaction.yaml

```
@@ -1,10 +1,22 @@
 ---
-operationId: authorize-transaction
-summary: Authorize approved transaction
-description: Authorize a previously approved transaction.
+operationId: approve-transaction-request
+summary: Request approve transaction
+description: |-
+  Request the next step in a 3D Secure flow.
 tags:
   - Transactions
-
+requestBody:
+  content:
+    application/json:
+      schema:
+        $ref: "../openapi.yaml#/components/schemas/TransactionApproveRequest"
+      examples:
+        Fingerprint step complete:
+          value:
+            
+        Challenge step complete:
+          value:
+            threeds_transaction_status: "N"
```

Is this `N` just an example value?

---

Yes, whatever is passed back from the issuer in the cRes should be passed in this field, however cardinal's examples show an N or Y in this field, hence the example value of 'N'

---

Cool. My only change would be to update this to:

```suggestion
            3d_secure_transaction_status: "N"
```